### ISSUE TITLE
TM API returns error: Your session has expired.  Please log in again.

### ISSUE THREAD

**Issue by CapicheDesign** [sleemanj/gogoTradeMe#6]
Hi there, I'm working with a legacy project that implements your module to display a list of TM job listings on the client's website. This has all been working well until recently when the API began returning the following error (from Chrome console):

```
ErrorDescription: "Your session has expired.  Please log in again."
Request: "https://api.trademe.co.nz/v1/Search/Jobs.json?member_listing=2180750&rows=200
```

Just wondered if you had any ideas why this might be occuring? The app on the TM account is enabled and approved. Thanks in advance for any tips!

**Comment by sleemanj** [comment]
I don't think that comes from my PHP bindings (gogoTradeMe), well, that is to say, the PHP bindings have no way to put anything like that in the Chrome console. 

I would suggest that you have some client side javascript which is making that request, and that maybe trademe is not allowing it any more (unauthenticated).

Of course it could be that your own code is using the PHP bindings to communicate to trademe and communicating back an error message to the console.... but it doesn't strike me as likely especially since gogoTradeMe uses xml as it's default transport not json (ie `Jobs.json` in your error).

**Comment by CapicheDesign** [comment]
Thanks, yes there is some client side JS that has been integrated with gogoTradeMe, what's puzzling is why it work fine for years and suddenly stop. Maybe a change in the TradeMe API I suppose, I have emailed them and will see what they come back with. Appreciate the response!
